docs/docker/readme.md

```diff
@@ -0,0 +1,8 @@
+# Utiliser r2gg avec Docker 
+
+Le dossier {{ '[docker]({}tree/{}/docker)'.format(repo_url, repo_branch) }} regroupe les différents fichiers permettant d'utiliser r2gg avec docker. 
+
+Les sous-dossiers sont ordonnés ainsi pour des raisons pratiques : 
+- {{ '[scripts]({}tree/{}/docker/scripts)'.format(repo_url, repo_branch) }} : Ce dossier regroupe des scripts utiles dans les différents docker. 
+- {{ '[config]({}tree/{}/docker/config)'.format(repo_url, repo_branch) }} : Ce dossier regroupe plusieurs fichiers de configurations qui se trouvent être communs aux autres sous-dossiers. 
+- {{ '[debian]({}tree/{}/docker/debian)'.format(repo_url, repo_branch) }} : Pour utiliser r2gg sur debian 10
```

Comment:
No need to use docker/debian/readme in repository. You can use the docs/docker/debian/readme.md file

Reply:
In original doc, there is a link to `docker/debian` folder, not the `readme.md`. What do you think is the best to link ? Folder or doc about debian docker image ?

Reply:
I think it's best to use the doc about debian docker image

Reply:
Updated !